sdk/cognitiveservices/azure-cognitiveservices-language-textanalytics/azure/cognitiveservices/language/textanalytics/aio/__init__.py

```diff
@@ -0,0 +1,375 @@
+            model_version=model_version,
+            show_stats=show_stats,
+            **kwargs
+        )
+        if response[0].is_error:
+            return process_single_error(response[0])  # DocumentError
+        return response[0]  # DocumentEntities
+
+
+async def single_recognize_linked_entities(
+        endpoint,  # type: str
+        credential,  # type: str
+        text,  # type: str
+        language="en",  # type: Optional[str]
+        show_stats=False,  # type: Optional[bool]
+        model_version=None,  # type: Optional[str]
+        **kwargs  # type: Any
+):
+    # type: (...) -> DocumentLinkedEntities
+    """Recognize linked entities from a well-known knowledge base
+    for a single document.
+
+    Returns a list of recognized entities with links to a
+    well-known knowledge base. See https://aka.ms/talangs for
+    supported languages in Text Analytics API.
+
+    :param str endpoint: Supported Cognitive Services endpoints (protocol and
+        hostname, for example: https://westus2.api.cognitive.microsoft.com).
+    :param credential: Credentials needed for the client to connect to Azure.
+        This can be the cognitive services subscription key or a token credential
+        from azure.identity.
+    :type credential: str or token credential
+    :param str text: The single string to recognize entities from.
+    :param str language: This is the 2 letter ISO 639-1 representation
+        of a language. For example, use "en" for English; "es" for Spanish etc. If
+        not set, uses "en" for English as default.
+    :param bool show_stats: If set to true, response will contain
+        document level statistics.
+    :param str model_version: This value indicates which model will
+        be used for scoring. If a model-version is not specified, the API
+        will default to the latest, non-preview version.
+    :return: An instance of DocumentLinkedEntities
+    :rtype: ~azure.cognitiveservices.language.textanalytics.DocumentLinkedEntities
+    :raises: ~azure.core.exceptions.HttpResponseError
+
+    .. admonition:: Example:
+
+        .. literalinclude:: ../samples/async_samples/sample_single_recognize_linked_entities_async.py
+            :start-after: [START single_recognize_linked_entities_async]
+            :end-before: [END single_recognize_linked_entities_async]
+            :language: python
+            :dedent: 8
+            :caption: Recognize linked entities in a single string.
+    """
+    doc = _validate_single_input(text, "language", language)
+    async with TextAnalyticsClient(endpoint, credential=credential, **kwargs) as client:
+        response = await client.recognize_linked_entities(
+            documents=doc,
+            model_version=model_version,
+            show_stats=show_stats,
+            **kwargs
+        )
+        if response[0].is_error:
+            return process_single_error(response[0])  # DocumentError
+        return response[0]  # DocumentLinkedEntities
```

Comment:
If you dont prefix by "type" mypy ignores it (I think :))

Reply:
I tried prefixing with type and mypy threw an error saying "misplaced type annotation". I'm happy to just leave the comments there for clarity.